include/edyn/math/geom.hpp

```diff
@@ -296,13 +296,14 @@ size_t intersect_line_aabb(const vector2 &p0, const vector2 &p1,
  */
 bool point_in_polygonal_prism(const std::vector<vector3> &vertices,
                               const std::vector<size_t> &indices,
-                              const vector3 &normal, const vector3 &point);
+                              const vector3 &normal, const vector3 &point) noexcept;
 
 template<size_t N>
-bool point_in_polygonal_prism(const std::array<vector3, N> &vertices,
-                              const vector3 &normal, const vector3 &point) {
+constexpr bool point_in_polygonal_prism(const std::array<vector3, N> &vertices,
+                              const vector3 &normal, const vector3 &point) noexcept {
+    
     const auto count = vertices.size();
-    EDYN_ASSERT(count > 2);
+    static_assert(count > 2);
```

Comment:
This is giving me errors such as "static_assert expression is not an integral constant expression", "initializer of 'count' is not a constant expression", so I guess I'll just use `N`.